assets/ExpressionMetadata.json

```diff
@@ -458,6 +458,18 @@
                 }
             ]
         },
+        {
+            "name": "listKeyValue",
+            "expectedUsage": "listKeyValue(resourceName\/resourceIdentifier, apiVersion, KeyLabelObject)",
+            "description": "Returns a value from the App Config store with Key/Label specified in the request. The KeyLabelObject parameters is a JSON object with the two properties \"key\" and \"label\" as required by the values in the store.",
+            "minimumArguments": 3,
+            "maximumArguments": 3,
+            "returnValueMembers": [
+                {
+                    "name": "value"
+                }
+            ]
+        },
         {
```

Comment:
We could also relatively easily add a field for a documentation link (https://docs.microsoft.com/en-us/rest/api/appconfiguration/configurationstores/listkeyvalue), if you think that makes sense.

Reply:
It's not doc'd yet (separate bug for that) but we could link here: https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions#resource-functions

We don't actually doc each function in the template docs - could reference the REST API though that's a little out of context.  Could also reference quickstart but that link would eventually get broken